docs/commands/kam_component_add.md

```diff
@@ -27,6 +27,8 @@ kam component add [flags]
   -h, --help                      help for add
       --interactive               If true, enable prompting for most options if not already specified on the command line
       --output string             Folder path to the Application to add the Component (default "./")
```

Comment:
For "./", let's be platform neutral.  Our CLI will work on Windows.  So use "." instead or mention default is the current folder.  There are other places where you introduce this, make it consistent across the board.